### PR TITLE
Add link to lima xbar plugin

### PR DESCRIPTION
**Pull request by unixorn**
- Add link to [lima-xbar-plugin](https://github.com/unixorn/lima-xbar-plugin)
- Sort adopters list

Closes https://github.com/lima-vm/lima/issues/253

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,9 @@ to Mac users, but Lima can be used for non-container applications as well.
 
 ## Adopters
 
-- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
+- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - Plugin for [xbar](https://xbarapp.com/) to create a Lima menu in your macOS menubar to start/stop your Lima VMs and see their running status.
```
Comment (jandubois) [inline README.md:40]: Seems slightly verbose compared to the other entries; it is kind of implied that this is about Lima. 😄 

```suggestion
- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - [xbar](https://xbarapp.com/) plugin to start/stop VMs from the menu bar and see their running status.
```

On second thoughts, this is just nit-picking, and the existing entry is fine too.

**Review by jandubois — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -36,8 +36,9 @@ to Mac users, but Lima can be used for non-container applications as well.
 
 ## Adopters
 
-- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
 - [Colima](https://github.com/abiosoft/colima): Docker (and Kubernetes) on macOS with minimal setup
+- [Lima xbar plugin](https://github.com/unixorn/lima-xbar-plugin) - [xbar](https://xbarapp.com/) plugin to start/stop VMs from the menu bar and see their running status.
+- [Rancher Desktop](https://rancherdesktop.io/): Kubernetes and container management to the desktop
```
Comment (AkihiroSuda) [inline README.md:41]: I don’t think the list has to be alphabetically sorted.

Also, could you squash commits?

Comment (jandubois) [reply README.md:41]: > I don’t think the list has to be alphabetically sorted.

You'll need to let @unixorn know which order you prefer. Is it by original release date or something else?

> Also, could you squash commits?

Yes, the change done via the Github website is also missing the DCO line.

Comment (unixorn) [reply README.md:41]: Squashed the commits and signed the new squashed one.

I'm happy to revert the sort to the original order. It doesn't have to be alphabetical, I just feel that makes it easier to find things - not an issue now with three items, but for the future.

**Review by AkihiroSuda — APPROVED** [review]
Added a follow-up commit

**Pull request merged**